.travis.yml

```diff
@@ -7,7 +7,10 @@ language: python
 # this ensures that the pipeline is triggered for internal pushes,
 # PRs from forks and pushes to existing PRs from forks
 if: (type == push) OR (type == pull_request AND fork == true)
-
+branches:
```

Comment:
Oh, this is why it didn't trigger a Travis run. Why did you add that? I think we would like to run tests always, for each PR. If you would like for specific tests (like integration tests), you should make use of different stages or a job matrix, then introduce conditions for branches separately for each stage/job. It should also be possible to use conditions on the exact commands, but the syntax is not great and it can quickly become unreadable for more complicated files. In any case, I don't see a good reason why we should not run the Travis CI for all PRs as well, the tests should be fast enough...

Reply:
(so I would just remove this block again unless there is a convincing reason to keep it)